lib/whitehall.rb

```diff
@@ -162,6 +162,10 @@ def self.organisations_in_tagging_beta
       YAML.load_file(Rails.root + "config/organisations_in_tagging_beta.yml")["organisations_in_tagging_beta"]
   end
 
+  def self.tagging_taxonomy_enabled?
```

Comment:
It wasn't moved, it was duplicated. I also don't understand why we're still using a feature flag here, because it should be visible on all environments.